p2p/src/message.rs

```diff
@@ -86,11 +86,11 @@ impl HeaderResponse {
 }
 
 #[derive(Debug, Encode, Decode, Clone, PartialEq, Eq)]
-pub struct BlockResponse {
+pub struct BlocksResponse {
```

Comment:
Am I the only one who thinks the pluralisation of this sounds strange? `BlocksResponse` doesn't really make sense

Reply:
No, I don't like it either but it was decided that we go with this

Reply:
It's plural because it's about multiple blocks.

Reply:
Could it be something like `BlockListResponse` or `BlockSetResponse`? I find `BlocksResponse` a bit weird too.

Reply:
I don't mind. Something like BlockListResponse is equally OK for me.

Reply:
Yeah fine by me

Reply:
I'd suggest the same for `HeadersResponse`

Reply:
I suppose we should also change the names of the corresponding requests (`BlockRequest` and `HeaderRequest`).